mobile/examples/Maui/MauiVisionSample/readme.md

```diff
@@ -0,0 +1,106 @@
+# ONNX Runtime MAUI Vision Sample
+
+The [MAUI Vision Sample](MauiVisionSample.sln) demonstrates the use of two different vision models from the [ONNX Model Zoo collection](https://github.com/onnx/models/tree/main), by a [MAUI](https://docs.microsoft.com/en-us/dotnet/maui/what-is-maui) app.
+
+## Overview
+The app enables you to take/pick a photo on the device or use a sample image to explore the following models.
+
+### [Mobilenet](https://github.com/onnx/models/tree/main/vision/classification/mobilenet)
```

Comment:
nit: should we use a permalink here too, like the Ultraface link?

Reply:
I went the other way and changed the link for ultraface so that if new versions of the model are added (e.g. one with the fixes we are manually making) they will show up. 

Cost is that if the page moves completely the link will break, but that seems less likely. e.g. for mobilenet new versions were added relatively recently.

Reply:
j